java/src/main/java/system/gameplay/roleplay/DiceRoller.java

```diff
@@ -5,26 +5,53 @@
 import java.util.List;
 import java.util.Random;
 
-public class DiceRoller {
+public final class DiceRoller {
 
-    private final Random random;
+    private static final Random random = new Random();
 
-    public DiceRoller() {
-        random = new Random();
+    public static List<Integer> rollAbilityScores() {
+        //TODO: (A) This line is here so the compiler won't complain, remove it when you add the functionality for this method
+        ArrayList<Integer> rollAbilityScoresList = new ArrayList<>();
+        int topThreeSixSidedDie = topThreeSixSidedDie();
+        for (int i = 1; i <= 6; i++) {
+                rollAbilityScoresList.add(topThreeSixSidedDie);
+                topThreeSixSidedDie = topThreeSixSidedDie();
+        }
+
+        // Sort scores to make it easier for player to keep track of how they're prioritizing their abilities
+        sortIntsHighToLow(rollAbilityScoresList);
+        return rollAbilityScoresList;
     }
 
-    public List<Integer> rollAbilityScores() {
-        //TODO: (A) This line is here so the compiler won't complain, remove it when you add the functionality for this method
-        return new ArrayList<>();
+    public static int roll6() {
+        return random.nextInt(6) + 1;
     }
 
-    public int roll20() {
-        // Returns a random int between 1 and 20
+    public static int roll20() {
         return random.nextInt(20) + 1;
     }
 
-    private List<Integer> sortIntsHighToLow(List<Integer> list) {
-        Collections.reverse(list);
-        return list;
+    public static int topThreeSixSidedDie() {
```

Comment:
I don't think that this will ever be used anywhere else, so we can go ahead and make it private.

Also, the name of this method doesn't follow the [Java Naming Conventions] (https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html) for methods. Method names should be verbs. In this case, I think `rollForScore` or something along those lines would be a decent name. You can add a [JavaDoc](https://www.baeldung.com/javadoc) style comment above this method and add an explanation of what the method does in there.